### ISSUE TITLE
The fourth stroke of the character 體 is not accepted.

### ISSUE THREAD

**Issue by peterolson** [chanind/hanzi-writer#163]
Try doing the stroke quiz on the demo page with the character 體.

https://hanziwriter.org/demo.html

The fourth stroke of 體, which is a short horizontal stroke, is not being accepted even if I draw exactly on top of the outline.

**Comment by chanind** [comment]
Weird, it seems to work when I try it:


![體_vid](https://user-images.githubusercontent.com/200725/75625457-1142fe00-5bb6-11ea-930c-f00bdec46509.gif)

**Comment by peterolson** [comment]
Aa now I understand the cause of the issue. Both the user and I assumed that the stroke would go from left-to-right. I guess technically it works as designed, although the right-to-left direction here is a bit unexpected, and it's easy to not notice that detail since it's such a short stroke. At least one user got stuck at this point and was unable to continue with the quiz because they could not get past this stroke, and I also got stuck reproducing it because I made the same mistaken assumption about the stroke direction.

I wonder if there's an elegant way to indicate to the user here that they wrote the stroke in the wrong direction?

**Comment by chanind** [comment]
Yeah that makes sense. The mistake highlight is supposed to indicate direction too, but it looks like it's too fast and hard to see for short strokes. Might need some more tweaking to make sure the direction is clearly visible in the highlight.
